compliance_checker/runner.py

```diff
@@ -16,7 +16,8 @@ class ComplianceChecker(object):
     the command line or can be used via import.
     """
     @classmethod
-    def run_checker(cls, ds_loc, checker_names, verbose, criteria, output_filename='stdout', output_format='stdout'):
+    def run_checker(cls, ds_loc, checker_names, verbose, criteria,
+                    skip_checks=[], output_filename='stdout', output_format='stdout'):
```

Comment:
I'm aware of the issue when mutating an array, but this code shouldn't be modifying the default arg.  I could do a check for `None` instead if you'd like.